python/sglang/srt/model_executor/model_runner.py

```diff
@@ -350,6 +367,21 @@ def init_flashinfer(self):
                         use_tensor_cores=use_tensor_cores,
                     )
                 )
+                
+    def init_double_sparsity_channel_config(self, selected_channel):
+        
+        selected_channel = "." + selected_channel + "_proj"
```

Comment:
Also add some end2end test under https://github.com/sgl-project/sglang/tree/main/test/srt, you can see this as an example https://github.com/sgl-project/sglang/blob/main/test/srt/test_torch_compile.py

Reply:
@merrymercy To run e2e results, it needs an offline calibration config. Can I put Llama config under `test` dir?

Reply:
yes, if it is a small text file